test/4_integration.js

```diff
@@ -73,14 +75,18 @@ describe('Integration tests', function() {
 	});
 
 	after(function() {
-		if (window.WEB_BUILD) {
+		if (window.WEB_BUILD && typeof branch._server.createScript.restore === 'funciton') {
```

Comment:
Nice catch! 
![image](https://cloud.githubusercontent.com/assets/10977790/9829967/c97f16ee-58d1-11e5-8353-35bc1a0b3635.png)

The reason it's passing the tests is that it's in after() so the function doesn't get restored when all the test cases are processed, but we no longer have more test cases by then. It's still a good idea to restore, though.

The reason it's passing lint is ... uh ... I turned it off because I committed very many times today. Turning it back on since we are close to finish!